tools/ci_build/github/azure-pipelines/linux-multi-gpu-tensorrt-ci-pipeline.yml

```diff
@@ -1,9 +0,0 @@
-jobs:
```

Comment:
Current this yaml file is not set up to a pipeline. If needed, we should rewrite this file with the same docker file as the other TRT pipelines.

Reply:
@HectorSVC set this up awhile back. can you provide context? Which pipeline was set hooked up to previously?

Reply:
Never mind. I just put one at: https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=169056&view=results